zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/group/PersistedGroup.java

```diff
@@ -23,6 +24,11 @@ public PersistedGroup() {
     declareProperty(groupKeyProp).declareProperty(nameProp);
   }
 
+  public void wrap(final GroupRecord group) {
+    groupKeyProp.setValue(group.getGroupKey());
+    nameProp.setValue(group.getName());
```

Comment:
🔧 The `getName` method makes a string from the buffer, you can use `getNameBuffer()` to directly set the name buffer